.github/actions/install-helm-chart/action.yml

```diff
@@ -0,0 +1,75 @@
+name: "Deploy Keptn on GH cluster"
+description: "Creates a Kind cluster and deploys Keptn"
+inputs:
+  kind-version:
+    required: false
+    description: "Version of kind that should be used"
+    default: "v0.18.0"
+  k8s-version:
+    required: false
+    description: "Kubernetes version that should be used"
+    default: "v1.27.3"
+  cluster-name:
+    required: false
+    description: "Name of the kind cluster"
+    default: "test-cluster"
+runs:
+  using: "composite"
+  steps:
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v5
+      with:
+        go-version: ${{ env.GO_VERSION }}
+        cache: true
+        cache-dependency-path: '**/go.sum'
+        check-latest: true
+
+    - name: Download artifacts
+      uses: actions/download-artifact@v3
+      with:
+        path: ~/download/artifacts
+
+    - name: "Create single kind Cluster"
+      uses: helm/kind-action@v1.8.0
+      with:
+        cluster_name: ${{ inputs.cluster-name }}
+        version: ${{ inputs.kind-version }}
+        node_image: "kindest/node:${{ inputs.k8s-version }}"
+        kubectl_version: ${{ inputs.k8s-version }}
+
+    - name: Import images in k3d
+      shell: bash
+      run: |
+        cd ~/download/artifacts
+        for image in $(ls | grep image.tar);
+        do
+          echo "Importing image: $image"
+          kind load image-archive $image/$image -n ${{ inputs.cluster-name }}
+        done
+
+    - name: Install lifecycle-toolkit with helm
+      env:
+        RELEASE_REGISTRY: "localhost:5000/keptn"
+      shell: bash
+      run: |
+        echo "Installing Keptn using helm"
+        helm version
+
+        helm repo add keptn "https://charts.lifecycle.keptn.sh"
+        helm repo update
+
+        for chart_dir in ./lifecycle-operator/chart \
+                ./metrics-operator/chart \
+                ./keptn-cert-manager/chart \
+                ./chart; do
+            cd "$chart_dir"
+            echo "updating charts for" $chart_dir
+            helm dependency update
+            helm dependency build
+            cd -  # Return to the previous directory
+        done
+
```

Comment:
these line do not seem needed here since yu use the default keptn repo to install the chart

Reply:
oh yeah, will remove these